ansible/playbook/roles/launcher/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
```

Comment:
This whole file seems out of place

Reply:
Oups. I will delete the content of the main.yml file

Reply:
:)